manager/Dockerfile

```diff
@@ -26,6 +29,15 @@ RUN --mount=type=cache,target=/var/lib/apt/lists,sharing=locked \
         libgeos-dev \
         libtool \
         autoconf \
+        ca-certificates \
```

Comment:
These should all be removed again in a later layer to keep the image size from getting big.